covid19-etl/etl/idph_facility.py

```diff
@@ -117,14 +119,15 @@ def parse_facility(self, date, facility):
         facility_name = facility["FacilityName"]
         confirmed_cases = facility["confirmed_cases"]
         deaths = facility["deaths"]
-        status = facility["status"]
+        status = facility.get("status", "Open")
 
         summary_location_submitter_id = format_submitter_id(
             "summary_location",
             {
                 "country": self.country,
                 "state": self.state,
                 "facility_name": facility_name,
+                "status": status,
```

Comment:
should this be reporting_org_status as well?
```suggestion
                "reporting_org_status": status,
```

Reply:
Actually, `format_submitter_id` doesn't care about the keys in dictionary, it's for nice way to specify the values for `submitter_id`. But it's nice to have it coherent. 👍

Reply:
oh right, woops